soc/silabs/silabs_s2/xg28/Kconfig.defconfig

```diff
@@ -5,6 +5,12 @@ if SOC_SILABS_XG28
 
 config NUM_IRQS
 	# must be >= the highest interrupt number used
-	default 79
+	default 78
+
+config PM
+	select UART_INTERRUPT_DRIVEN
+
+config GPIO_GECKO
```

Comment:
This should be removed. It's the cause of the CI failure, Series 2 devices now use a new GPIO driver.